src/Netflix/Netflix.ts

```diff
@@ -0,0 +1,875 @@
+      },
+      {
+        name: "Seek Behind By Second",
+        match: [
+          "skip behind # [minute/minutes]",
+          "skip behind # [minute/minutes] # [second/seconds]"
+        ],
+        pageFn: (_: string, minute: number, second = 0) =>
+          sendMessage({
+            key: "skip",
+            sub: { key: "behind", duration: (60 * minute + second) * 1000 }
+          }),
+        normal: false
+      },
+      {
+        name: "Seek Behind By Second",
+        match: ["skip behind # [second/seconds]"],
+        pageFn: (_: string, second: number) =>
+          sendMessage({
+            key: "skip",
+            sub: { key: "behind", duration: second * 1000 }
+          }),
+        normal: false
+      },
+
+      {
+        name: "Watch By Title",
+        match: ["watch *"],
+        pageFn: (_: string, query: string) =>
+          sendMessage({
+            key: "watch",
+            sub: {
+              key: "ask-videos",
+              query
+            }
+          }),
+        normal: false
+      },
+      {
+        name: "Watch Random Show",
+        match: ["random"],
+        pageFn: (_: string) => sendMessage({ key: "watchRandom" }),
+        normal: false
+      },
+      {
+        name: "Search Show",
+        match: ["search *"],
+        pageFn: (_: string, title: string) => navigateToSearch(title),
+        normal: false
+      }
+    ]
+  }
+};
+
+/**
+ * Receive Message
+ */
+(() => {
+  window.addEventListener("message", ev => receiveMessage(ev.data));
+  const receiveMessage = (messageStr: string) =>
+    consumeMessageStringAsCommand(messageStr, FROM_PAGE_PROOF_KEY, command => {
+      switch (command.key) {
+        case "changeText":
+          return handleChangeTextAnswer(command);
+        case "changeAudio":
+          return handleChangeAudioAnswer(command);
+        case "watch":
+          return handleWatchAnswer(command);
+      }
+    });
+})();
+
+const handleWatchAnswer = async (command: WatchCommand) => {
+  const { sub } = command;
+  if (sub.key !== "answer-matches") return;
+  const videos = sub.videos;
+
+  // This exact title matching video works well because LipSurf scans for ariaLabel which
+  // actually matches the videos title queried
+  const exactVideo = videos.find(video => video.title === sub.query);
+  if (exactVideo) {
+    return navigateToWatch(exactVideo.videoId);
+  }
+
+  // Filter out videos which title are empty string
+  const filteredVideos = videos.filter(
+    video => !!stripNonAlphaNumericAndTrim(video.title)
+  );
+
+  const strippedQuery = stripNonAlphaNumericAndTrim(sub.query);
+
+  /**
+   * Sort descending. fuzzyHighScore somehow does not return the same value
+   * as these sequence below. If it turns out a bug, then
+   * TODO: replace these sequence with fuzzyHighScore
+   */
+  const results = (
+    await Promise.all(
+      filteredVideos.map(video =>
+        PluginBase.util.fuzzyHighScore(strippedQuery, [video.title], 0, true)
```

Comment:
you shouldn't need to strip the query, that string gets preprocessed the same way the sources do.

Reply:
Ok